src/Plugin/Visualisation/Style/VisualisationStyleBase.php

```diff
@@ -41,7 +41,13 @@ public function __construct(array $configuration, $plugin_id, $plugin_definition
    * {@inheritdoc}
    */
   public function getConfiguration() {
-    return NestedArray::mergeDeep($this->defaultConfiguration(), $this->configuration);
+    $configuration = NestedArray::mergeDeep($this->defaultConfiguration(), $this->configuration);
+
+    // Allow other modules to alter style configuration before build via
+    // hook_dvf_style_configuration_alter().
+    \Drupal::moduleHandler()->alter('dvf_style_configuration', $configuration, $this);
```

Comment:
Can we please inject `module_handler` via the service container?

Reply:
I caved... updated